src/lib/wasm/RaygunStore.ts

```diff
@@ -406,8 +408,9 @@ class RaygunStore {
         let conversations: { inner: wasm.Conversation }[] = await raygun.list_conversations()
         let handlers: { [key: string]: Cancellable } = {}
         for (let conversation of conversations) {
-            let handler = await this.createConversationEventHandler(raygun, conversation.inner.id())
-            handlers[conversation.inner.id()] = handler
+            //typescript thinks id is a function, but in the browser it is not.
+            let handler = await this.createConversationEventHandler(raygun, conversation.inner.id)
+            handlers[conversation.inner.id] = handler
```

Comment:
Are these marked as getters?

Reply:
It isn't. Not sure why is being treated as one. We should probably set it as getter in warp so that at least typescript will see it as correct.

Reply:
Yea. We could probably do that in a separate PR